### PR TITLE
Posts and Text Updates

### PR DESCRIPTION
**Pull request by bkesecker**
Some basic text updates.  Some of these changes are already in Discord's version.  It will be easy for staff to update Discord with changes here since GitHub makes them easy to see.

### CODE REVIEW

File: api/v1/config.js [hunk 1]
```diff
@@ -1,11 +1,32 @@
 module.exports = {
   posts: [
     {
-      date: '2020-06-14T00:00:00.000Z',
+      date: '2020-06-20T09:35:00.000Z',
```
Comment (slassen) [inline api/v1/config.js:4]: Probably best to not change dates once they have been in prod for a while even if they were wrong. For people that index things mentally.

**Review by slassen — COMMENTED** [review]
We should either always add new entries to the top of the JSON file or make the parsing component reverse the array or parse array to sort by the dates so it displays the newest first. Also, one inline comment. Don't need to change it, just for future consideration.

File: api/v1/config.js (hunk 1 above)
Comment (bkesecker) [reply api/v1/config.js:4]: When publishing the website, I did not know what day it would actually go live, this reflects the actual date.

**Comment by bkesecker** [comment]
> We should either always add new entries to the top of the JSON file or make the parsing component reverse the array or parse array to sort by the dates so it displays the newest first. Also, one inline comment. Don't need to change it, just for future consideration.

Posts added are historical.  I would expect new posts to be at the top, but sorting sounds like a good feature.

File: api/v1/config.js [hunk 2]
```diff
@@ -26,27 +47,23 @@ module.exports = {
...
         'Character names that depict cheating, could be considered sexually explicit, or any names that could be disallowed on retail are NOT allowed.',
-        'Use of or advertisement of any cheat program/addon or a non-approved addon/program on Eden or its media. There will be an allowed list and a non-allowed list. Those not in the approved list are discouraged from use as you may be penalized for using them. Use at your own risk. Those that are similar to a forbidden program are most likely forbidden as well.',
+        'Use of, or advertisement of, any cheat program/addon or a non-approved addon/program on Eden or its media is NOT allowed. There will be an allowed list and a non-allowed list. Those not in the approved list are discouraged from use as you may be penalized for using them. Use at your own risk. Those that are similar to a forbidden program are most likely forbidden as well.',
         'GM interaction is limited here but if a GM messages you in game and you are active, respond within a reasonable time. Ignoring a GM is likely to get you jailed as we will likely think you are afk botting.',
       ],
```
Comment (slassen) [inline api/v1/config.js:68]: Ah ha. Understood. I didn’t actually look at the date.

**Review by slassen — APPROVED** [review]
Ah ha. Understood. I didn’t actually look at the date.

**Pull request merged**